packages/nuxt/src/app/components/nuxt-island.ts

```diff
@@ -50,7 +50,8 @@ export default defineComponent({
     const error = ref<unknown>(null)
     const config = useRuntimeConfig()
     const nuxtApp = useNuxtApp()
-    const hashId = computed(() => hash([props.name, props.props, props.context, props.source]))
+    const propsForHash = props.props ? Object.fromEntries(Object.entries(props.props).filter(([key]) => !key.startsWith('data-v-'))) : {}
```

Comment:
This won't be reactive, you need to move it into the computed or to make it computed

Reply:
Good point